src/Faker/Provider/sv_SE/Person.php

```diff
@@ -20,97 +20,32 @@ class Person extends \Faker\Provider\Person
arie','Evelina','Evelyn','Evy','Ewa','Fanny','Florence','Fredrika','Frida','Frideborg','Gabriella','Gerd','Gerda','Gertie','Gertrud','Gisela','Greta','Gudrun','Gull','Gullan','Gullbritt','Gulli','Gullvi','Gully','Gull-Britt','Gun','Gunborg','Gunbritt','Gunda','Gunhild','Gunilla','Gunn','Gunnel','Gunni','Gunvor','Gun-Britt','Gurli','Gustava','Gärd','Görel','Göta','Hanna','Harriet','Hedvig','Helen','Helén','Helena','Helene','Heléne','Helfrid','Helga','Helmi','Helny','Henny','Henrietta','Henriette','Herta','Hilda','Hildegard','Hildur','Hillevi','Hilma','Hjördis','Hulda','Ida','Ines','Inez','Inga','Ingalill','Inga-Britt','Inga-Lena','Inga-Lill','Inga-Lisa','Inga-Maj','Ingbritt','Ingeborg','Ingegerd','Ingegärd','Ingela','Inger','Ingrid','Ingvor','Ing-Britt','Ing-Mari','Ing-Marie','Iréne','Irene','Iris','Irma','Isabella','Jane','Janet','Jeanette','Jenny','Jessica','Johanna','Josefina','Judit','Judith','Julia','Juliana','Justina','Kaarina','Kajsa','Karin','Karina','Karla','Karola','Karolina','Katarina','Katharina','Katrin','Katrina','Kersti','Kerstin','Klara','Konstantia','Kornelia','Kristin','Kristina','Kristine','Laila','Laura','Leila','Lena','Leontina','Liisa','Lilian','Lill','Lillemor','Lillian','Lilly','Linda','Linnéa','Linnea','Lisa','Lisbet','Lisbeth','Liselott','Liselotte','Lise-Lott','Lise-Lotte','Lizzie','Lola','Louise','Lovisa','Lucia','Lydia','Madeleine','Madelene','Magda','Magdalena','Magnhild','Maj','Maja','Majbritt','Majken','Majlis','Majvor','Maj-Britt','Maj-Lis','Malin','Malvina','Margaret','Margareta','Margareth','Margaretha','Margit','Margita','Margot','Margret','Margreta','Mari','Maria','Mariana','Mariann','Marianne','Marie','Mariette','Marie-Louise','Marika','Marina','Marion','Marit','Marita','Mari-Ann','Marja','Marjatta','Marlene','Marta','Martha','Martina','Mary','Mathilda','Matilda','Maud','May','Mia','Mildred','Mimmi','Mirjam','Mona','Monica','Monika','Märit','Märta','Märtha','Naemi','Naima','Nancy','Nanna','Nanny','Natalia','Nelly','Nina','Nora','Olga','Olivia','Ottilia','Paula','Paulina','Pauline','Pernilla','Petra','Petronella','Pia','Ragna','Ragnhild','Rakel','Rebecka','Regina','Renée','Rigmor','Rita','Rosa','Rose','Rose-Marie','Rosita','Ros-Mari','Ros-Marie','Runa','Rut','Ruth','Sabina','Saga','Sally','Sara','Selma','Serafia','Sibylla','Sigbritt','Signe','Signhild','Sigrid','Siri','Siv','Sofi','Sofia','Sofie','Solbritt','Solveig','Solvig','Sonja','Stina','Susann','Susanna','Susanne','Suzanne','Svea','Sylvia','Synnöve','Syster','Tea','Tekla','Terese','Teresia','Therése','Therese','Theresia','Thyra','Tina','Tora','Torborg','Tove','Tyra','Ulla','Ulla-Britt','Ulla-Britta','Ulrica','Ulrika','Ursula','Valborg','Vanja','Vega','Vendela','Vendla','Vera','Veronica','Veronika','Victoria','Viktoria','Vilhelmina','Vilma','Viola','Virginia','Vivan','Viveca','Viveka','Vivi','Vivian','Viviann','Vivianne','Vivi-Ann','Vivi-Anne','Wilhelmina','Ylva','Yvonne','Åsa','Åse','Elin ','Ida ','Josefin','Emelie','Amanda ','Sandra','Rebecca','Isabelle'
```

Comment:
what did you change? The diff is impossible to read, please respect the formatting (one line per capital letter)